R/derive_params_growth_age.R

```diff
@@ -283,6 +283,10 @@ derive_params_growth_age <- function(dataset,
         ) %>%
         select(-c(P95, Sigma))
     }
+    else {
+      add_pctl <- add_pctl %>%
+        mutate(AVAL = pnorm(AVAL)*100)
```

Comment:
@rossfarrugia I originally had the code set up was 

1. calculate z-score
2. use pnorm to calculated percentile
3. if bmi correction, use AVAL to recalculated appropriate percentile (this is incorrect cuz aval is currently already in percentile form 

so I modified it to 
1. get z-score
2. calculate bmi correction for percentile
3. !!! forgot to still calculate the percentile (This is what the code is now doing)

Reply:
that said, the numbers for percentile are finally looking correct again, I was so focused on checking that the bmi correction was doing done properly I forgot to check that it was running normally again